### DISCUSSION TITLE
possible to farm over cellular network?

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#6340] [Q&A]
Since you cannot do port forwarding over cellular network nominally, would you still be able to run a full node and do the farming? It seems that since 1.1.6, you can perfectly sync fine using cellular network.

**Comment by room101-dev** [comment]
Sure no problem, just connect the drive containing 'plots' to your computer running the miner ( solo and/or hpool ), then run the miner; Then place the mobile phone near the device and tell everybody you 'mining on the phone'

**Listen right now, under the new 5 sec rule, you can't even use USB or LAN anymore to 'farm',** so the answer is like I said above you can place your phone near the system running the miner, and connect the string to your phone and tell everybody your farming on your cellular.

Yep,  the phone company must love you running a full node at 5gb and doing the full-sync on a mobile, last I looked the chia code didn't even sync

I heard about a guy who rides a bicycle and farms by reading the HDD like braile and sending the data via his bike to the mining-pool at chia-net.

I suggest if you really want a good time, run the bitcond full-node at 600GB on your mobile device, then the phone company love you long time.
